app/views/projects/show.html.erb

```diff
@@ -38,11 +38,16 @@
             <td class="type" onclick="show(this, '<%= name %>');"><%= data_type.name %></td>
             <% data_type.data_values.includes(:project_year).sort_by_year.each do |data_value| %>
               <% if @project.project_years.find_by_id(data_value.project_year.id) %>
-                <td class="data-value" data-id=<%= data_value.id %> contentEditable="true" data-input-formula="<%= data_value.input_formula %>" onclick="showFormula(this, '<%= name %>');">
+                <td class="data-value" data-id=<%= data_value.id %> contentEditable="true" data-input-formula="<%= data_value.input_formula %>" data-type="<%= data_value.data_type.input_type%>" onclick="showFormula(this, '<%= name %>');">
                 <% if data_value.value %>
-                  <%= data_value.value %></td>
+                  <% @val = data_value.value %>
```

Comment:
better way to do this is to write a method in the DataValue model, that will run an "or" between formula_value and value, so we can just reference that.